eslint.config.mjs

```diff
@@ -0,0 +1,56 @@
+// @ts-check
+import eslint from '@eslint/js';
+import eslintPluginAstro from 'eslint-plugin-astro';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+	{
+		ignores: ['**/dist', '**/node_modules', '**/.astro', '**/.github', '**/.changeset'],
+	},
+
+	eslint.configs.recommended,
+	...tseslint.configs.recommended,
+
+	{
+		languageOptions: {
+			ecmaVersion: 'latest',
+			sourceType: 'module',
+			parser: tseslint.parser,
+		},
+		plugins: {
+			'@typescript-eslint': tseslint.plugin,
+		},
```

Comment:
I don't think the config enables any type-checked rules, and this also does not use the legacy format which required the `plugins` field but this is [not the case anymore](https://typescript-eslint.io/getting-started/typed-linting) with a flat config, so I think this entire section could be omitted.

Reply:
Ah, very possible, thank you for the feedback! I found the migration guidance pretty overwhelming and hard to piece together across the different plugins. Will try omitting this.

Reply:
Yeah, doesn’t seem to be needed testing locally — neither were a couple of the rules I had there based on the old config, so I removed those too. We can always add things back if we hit problematic lint errors later.